secret-script.py

```diff
@@ -1,64 +1,82 @@
-# Lint as: python3
-"""TODO(shanefu): DO NOT SUBMIT without one-line documentation for secret-script.
+#!/usr/bin/env python3
 
-TODO(shanefu): DO NOT SUBMIT without a detailed description of secret-script.
-"""
-'''
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import google_type_annotations
-from __future__ import print_function
-'''
-from absl import app
-from absl import flags
-
-FLAGS = flags.FLAGS
-
-flags.DEFINE_boolean('k2g', False, 'Secret sync from Kubernetes to Gcloud')
-flags.DEFINE_boolean('g2k', False, 'Secret sync from Kubernetes to Gcloud')
-flags.DEFINE_boolean('create', None, 'Create new secret with secret_id')
-flags.DEFINE_boolean('delete', None, 'Delete secret with secret_id')
-flags.DEFINE_boolean('get', None, 'Get secret with secret_id')
-flags.DEFINE_string('file', None, 'Create or update from file')
-flags.DEFINE_string('secret_id', None, 'The string id of the secret')
-
-flags.mark_flag_as_required("secret_id")
 
 from gcloud_utils import *
 from k8s_utils import *
+import argparse
+import sys
+
+def parse_args(args):
+	parser = argparse.ArgumentParser()
+	parser.add_argument('action', help='Options: create, get, update, or delete')
+	parser.add_argument('--secret_id', help='The id of the secret')
+	parser.add_argument('--file', help='The yaml file containing the secret')
+	parser.add_argument('--direction', help='Options: k2g or g2k')
+	return parser.parse_args(args)
```

Comment:
FWIW I more commonly see `parser.parse_args()` which eliminates some complexity in the function signature (you could just call `main(parse_args())` instead of passing in a slice, for example)

Reply:
Didn't notice this. You're right.
Fixed.